src/prefect/environments/execution/dask/k8s.py

```diff
@@ -423,7 +423,7 @@ def _populate_scheduler_spec_yaml(
 
         # Required Cloud environment variables
         env_values = [
-            {"name": "PREFECT__CLOUD__GRAPHQL", "value": prefect.config.cloud.graphql},
+            {"name": "PREFECT__CLOUD__GRAPHQL", "value": prefect.config.cloud.api},
```

Comment:
Why still set the env var at all, if nothing points at it? Should this be `PREFECT__CLOUD__API`?

Reply:
Very good point! There were a lot of values to update, and I got caught up in chugging through them and missed this. Let me confirm there are no additional ones like this.

Reply:
However, -- is this a breaking change? If my agent is running 0.9.2+ but my docker container has 0.9.1-, will it still run without the old value?

Reply:
Looking at some of our `deploy_flow` methods, I think it could be. Care to opine, @cicdw / @joshmeek ?

Reply:
We should make sure to QA this but I don't think this will be a breaking change.  We have been interpolating the `graphql` route from the `api` config setting [since 0.5.0](https://github.com/PrefectHQ/prefect/commit/c9081d18feee2a352a26a44787c1814832c0766f) and so setting the API config should be sufficient for Flows running old versions as well.